README.md

```diff
@@ -108,24 +108,24 @@ you'll need to do the following:
     - Periodic job run on a timed basis
     - You can find more prowjob definitions at [how-to-add-new-jobs](prow#how-to-add-new-jobs)
   - Please utilize the [podutils](prow/pod-utilities.md#how-to-configure) to create modern prowjobs!
-  - Scenario args: (if you are using [bootstrap.py](jenkins/bootstrap.py) instead of [podutils](prow/pod-utilities.md))
-    - [Scenarios](scenarios) are python wrappers used by our entry point script [bootstrap.py](jenkins/bootstrap.py).
-    - You can append scenario/kubetest args inline in your prowjob definition, example:
+    A simple sample job uses podutil looks like:
     ```yaml
-      - name: foo-repo-test
-        interval: 1h
-        agent: kubernetes
-        spec:
-          containers:
-          - image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
-            args:
-            - --repo=github.com/org/repo
-            - --timeout=90
-            - --scenario=execute
-            - --
-            - make
-            - test
+    - name: foo-repo-presubmit-test
+      decorate: true
+      spec:
+        containers:
+        - image: where/your/image/is
```

Comment:
maybe an example image or something? this is not a very typical looking docker image name..

Reply:
sure, promoted kubekins